tool/docs/common/Class.kt

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.vaticle.typedb.client.tool.doc.common
+
+data class Class(
+    val name: String,
+    val anchor: String? = null,
+    val enumConstants: List<EnumConstant> = listOf(),
+    val description: List<String> = listOf(),
+    val examples: List<String> = listOf(),
+    val fields: List<Variable> = listOf(),
+    val methods: List<Method> = listOf(),
+    val packagePath: String? = null,
+    val superClasses: List<String> = listOf(),
+) {
+    fun toAsciiDoc(language: String): String {
+        var result = ""
```

Comment:
I think we should create something slightly more readable/scalable to build ascii doc in a consistent way. Let's talk about it when you get to here